external_deps/build.sh

```diff
@@ -465,6 +464,7 @@ build_wasisdk() {
 		local WASISDK_PLATFORM=linux
 		;;
 	esac
+	local WASISDK_VERSION_MAJOR="$(echo "${WASISDK_VERSION}" | cut -f1 -d'.')"
```

Comment:
What's the point of having wasm related things in master since there is currently *no* support for it? Does not it just bloat the system?

Reply:
It's not included in the prebuilt packages, so it does not bloat the system.